### PR TITLE
Upgrade plugin to work with Kibana 7.10+

### PR DESCRIPTION
**Pull request by risdenk**
Addresses #48

Example of this built for testing: https://github.com/risdenk/analyze-api-ui-plugin/releases/tag/v7.16.2-alpha5

### CODE REVIEW

File: kibana.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "id": "analyzeApiUi",
+  "version": "7.16.2",
+  "kibanaVersion": "kibana",
+  "owner": {
+    "name": "risdenk"
```
Comment (risdenk) [inline kibana.json]: This needs to be updated.

File: kibana.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "id": "analyzeApiUi",
+  "version": "7.16.2",
```
Comment (risdenk) [inline kibana.json:3]: This should be updated?

File: package.json [hunk 3]
```diff
@@ -1,36 +1,10 @@
 {
   "name": "analyze_api_ui",
-  "version": "7.6.2",
-  "description": "UI for elasticsearch analyze API",
-  "main": "index.js",
-  "kibana": {
-    "version": "7.6.2",
-    "templateVersion": "1.0.0"
-  },
+  "version": "7.16.2",
```
Comment (risdenk) [inline package.json:3]: This should be updated?

**Comment by johtani** [comment]
Hi @risdenk, 
Thanks for your contribute!
I'll check this and learn new Kibana plugin API from your PR :)
Stay tuned.

File: kibana.json (hunk 1 above)
Comment (johtani) [reply kibana.json]: Please do...

File: kibana.json (hunk 2 above)
Comment (johtani) [reply kibana.json:3]: It's OK. I'll update after merging this PR

File: package.json (hunk 3 above)
Comment (johtani) [reply package.json:3]: It's OK. I'll update after merging this PR

File: kibana.json (hunk 1 above)
Comment (risdenk) [reply kibana.json]: Done in 3aeb265

**Comment by johtani** [comment]
Thanks @risdenk !
I found some bugs...
I'll fix and release soon :)

**Comment by risdenk** [comment]
Thanks @johtani! Yea I'm sure there are a few bugs. It worked for the quick port. Figured will continue to improve :)

**Pull request merged**